openbb_platform/providers/intrinio/openbb_intrinio/models/income_statement.py

```diff
@@ -95,30 +100,28 @@ def transform_data(
         """Return the transformed data."""
 
         transformed_data = []
+        data_key = f"{query.type}_financials"
+        tag_key = "xbrl_tag" if query.type == "reported" else "data_tag"
 
         for item in data:
             sub_dict = {}
 
-            if "reported_financials" in item:
-                key = "reported_financials"
-                sub_tag = "xbrl_tag"
-            elif "standardized_financials" in item:
-                key = "standardized_financials"
-                sub_tag = "data_tag"
-
-            for sub_item in item[key]:
-                try:
-                    sub_dict[sub_item[sub_tag]["tag"]] = int(
-                        sub_item[sub_tag]["factor"] + str(sub_item["value"])
-                    )
-                except (ValueError, KeyError):
-                    sub_dict[sub_item[sub_tag]["tag"]] = int(sub_item["value"])
+            for sub_item in item[data_key]:
+                field_name = alias_generators.to_snake(sub_item[tag_key]["name"])
+                sub_dict[field_name] = float(sub_item["value"])
 
             sub_dict["date"] = item["fundamental"]["end_date"]
             sub_dict["period"] = item["fundamental"]["fiscal_period"]
             sub_dict["cik"] = item["fundamental"]["company"]["cik"]
             sub_dict["symbol"] = item["fundamental"]["company"]["ticker"]
 
+            # Intrinio does not return Q4 data in reported_financials endpoint
```

Comment:
This is not true these both return data:

```
https://api-v2.intrinio.com/fundamentals/JPM-income_statement-2022-Q4/standardized_financials?
https://api-v2.intrinio.com/fundamentals/AAPL-income_statement-2022-Q4/standardized_financials?
```

Reply:
It says `reported_financials` also in the comment.

Reply:
They are all accepted through the same endpoint.

![Screenshot 2023-10-19 at 6 52 17 PM](https://github.com/OpenBB-finance/OpenBBTerminal/assets/85772166/a1861f6b-f0e7-4465-8622-962541de192d)